hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py

```diff
@@ -179,25 +179,23 @@ def _theoretical_hours(self, employee, date):
         """Get theoretical working hours for the day where the check-in is
         done for that employee.
         """
+        date_date = fields.Datetime.from_string(date)
         if not employee.resource_id.calendar_id:
             return 0
-        tz = employee.resource_id.calendar_id.tz
```

Comment:
Why removing timezone info?

Reply:
Because 'resource.calendar' object has no attribute 'tz' in 11.0

Reply:
Yeah, but the tz should reside in other part (user, employee, resource...)